tests/test_utils/test_opt.py

```diff
@@ -230,6 +235,29 @@ def cost_fn():
     assert np.allclose(np.sinh(circ.trainable_parameters["euclidean"][2]) ** 2, 1, atol=1e-2)
 
 
+def test_parameter_passthrough():
+    """Same as the test above, but with param passthrough"""
+    tf.random.set_seed(137)
+    r = np.arcsinh(1.0)
+    par = Parametrized(
+        r=math.new_variable(r, (0.0, None), "r"),
+        phi=math.new_variable(np.random.normal(), (None, None), "phi"),
```

Comment:
Should this variable be parametrized with hypothesis instead of defined here?

Reply:
> Great @ziofil! I just suggest to extend a bit the changelog to better describe the change.
> 
> On a more general point, I wonder if parameters should be better represented by classes themselves, in that way they can self-contain all the relevant information and avoid having to include conditional logic when dealing with them — a parameter object would know if it is trainable or constant, if already defined as a tensor in the backend or not, etc. 🤔

Yeah that's a good idea. I've thought about implementing a Parameter class when I wanted to track the history of each parameter and plot it, but haven't found time to work on it yet : /

Reply:
Thanks for the change in the changelog, it looks great!